docs/docs/blocks/!/AnyMore.md

```diff
@@ -0,0 +1,36 @@
+# AnyMore
+
+```clojure
+(AnyMore
+  :Value [(Any)]
+)
+```
+
+## Definition
+
+
+## Parameters
+| Name | Type | Default | Description |
+|------|------|---------|-------------|
+| Value | `[(Any)]` | `None` | The value to test against for equality. |
```

Comment:
I think should create a issue to track the `Type` further parsing. But for now we can skip it.

Reply:
Need some utilities to manipulate strings (skip, substr, remove, concat...).

Reply:
Ok, that's also good issues material